.travis.yml

```diff
@@ -12,41 +10,41 @@ cache:
   directories:
     # we also need to cache folder with Cypress binary
     - ~/.cache
-env:
-  global:
-    - WEBSERVICE_VERSION="1.6.0-beta.4"
 addons:
   firefox: "latest"
   postgresql: "9.6"
   chrome: stable
 
 install:
-  - ./scripts/install-travis-script.sh
+  - CI=true npm ci #CI=true to be consistent with compose_setup where it must be manually set
```

Comment:
Better, but I think I'd also like to know what it does (in code comment) as opposed to it simply being consistent

Reply:
it'll be in the compose_setup instead since it isn't entirely necessary here